string voice_selection
bool control_voice
